solace-spring-cloud-starters/solace-spring-cloud-stream-starter/README.md

```diff
@@ -111,14 +111,24 @@ For general binder configuration options and properties, refer to the [Spring Cl
 
 The following properties are available for Solace consumers only and must be prefixed with `spring.cloud.stream.solace.bindings.<channelName>.consumer.`.
 
-See [SolaceConsumerProperties](../../solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/properties/SolaceConsumerProperties.java) for the most updated list.
+See [SolaceCommonProperties](../../solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/properties/SolaceCommonProperties.java) and [SolaceConsumerProperties](../../solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/properties/SolaceConsumerProperties.java) for the most updated list.
 
 <dl>
     <dt>prefix</dt>
     <dd>
         <p>Naming prefix for all topics and queues.</p>
         <p>Default: ""</p>
     </dd>
+    <dt>provisionDurableQueue</dt>
+    <dd>
+        <p>Whether to provision durable queues for non-anonymous consumer groups. This should only be set to false if you have externally pre-provisioned the required queue on the message broker. Typically, this queue's name should have a format of `{prefix}{destination}.{group}`.</p>
+        <p>Default: true</p>
+    </dd>
+    <dt>addDurableQueueSubscription</dt>
```

Comment:
Consider renaming this to start with "provision" like the other 2 related config options. 
Maybe "provisionTopicSubsToDurableQueue" ?

Reply:
I kind of wanted it to be consistent with the `queueAdditionalSubscriptions` property. Unless you think its fine if they differ, how about `provisionSubscriptionsToDurableQueue` or `provisionDurableQueueSubscriptions`?

Reply:
that makes sense. I'm leaning towards `provisionSubscriptionsToDurableQueue`